.history/src/index_20181210141750.js

```diff
@@ -0,0 +1,161 @@
+import reverseArrayInPlace from './reverseArrayInPlace'
```

Comment:
You should add `.history/` to your `.gitignore_global` file on your machine.

Please do that, then run `git rm -rf .history` and update this branch with these files removed :)